### PR TITLE
Migrating to aws-sdk v3 and only including necessary gems

### PR DESCRIPTION
**Pull request by jesseadams**
The aws-sdk gem pulls in 174 gems when we only need 1.

### CODE REVIEW

**Comment by ChrisPetr0** [comment]
should we update trollop to the new name since it throws that warning all the time now?

File: crossing.gemspec [hunk 1]
```diff
@@ -29,7 +29,7 @@ DESC
   s.add_development_dependency('rubygems-tasks')
   s.add_development_dependency('simplecov')
 
-  s.add_runtime_dependency('aws-sdk', '>= 2.0.0')
-  s.add_runtime_dependency('trollop', '~ 2')
+  s.add_runtime_dependency('aws-sdk-s3')
```
Comment (jeffb4) [inline crossing.gemspec]: don't we need the kms gem as well?

Comment (jesseadams) [reply crossing.gemspec]: S3 pulls it in automatically according to `bundle install`

Comment (jeffb4) [reply crossing.gemspec]: Per Jesse, kms is pulled in transitively

File: .ruby-version [hunk 2]
```diff
@@ -1 +1 @@
-2.2.9
```
Comment (jeffb4) [inline .ruby-version]: Please bump to 2.3.0 as this effectively removed sdk v2 support, thanks

Comment (jeffb4) [reply .ruby-version]: oops wrong string

**Pull request merged**